app/api/models/errors/CommonMtdErrors.scala

```diff
@@ -22,8 +22,8 @@ import play.api.http.Status._
 
 // Format Errors
 object NinoFormatError extends MtdError("FORMAT_NINO", "The NINO format is invalid", BAD_REQUEST)
-
-object TaxYearFormatError extends MtdError("FORMAT_TAX_YEAR", "The taxYear format is invalid", BAD_REQUEST)
+object TaxYearFormatError extends MtdError("FORMAT_TAX_YEAR", "The provided tax year is invalid", BAD_REQUEST)
+//object TaxYearFormatError extends MtdError("FORMAT_TAX_YEAR", "The taxYear format is invalid", BAD_REQUEST)
```

Comment:
Is this comment necessary? 
```suggestion

```

Reply:
Good spot. This was commented out while testing something. Have changed it back, thanks.